packages/jellyfish-json/src/remap.ts

```diff
@@ -24,14 +24,20 @@ enum MappingAction {
  * @param precision PrecisionMapping is a key value pair to allow revive value type
  */
 export function remap (text: string, precision: PrecisionMapping): any {
-  const losslessObj = parse(text)
+  const { result: losslessObj } = parse(text)
```

Comment:
FYI, the ApiClient.call contains the `result` in resp
and gonna add the `result` in resp manually as well after remapped in ln: 40